Javadoc error

When `mvn clean package 2>&1 > log` is run, we get

```console
[ERROR] MavenReportException: Error while generating Javadoc:
Exit code: 1
/private/var/folders/k7/dppp4nxs0bl74h2_z3mkg_gr0000gn/T/docker_tmp_dir.1kxU9J7qNL/powerauth-push-server/powerauth-push-server/src/main/java/io/getlime/push/repository/model/Platform.java:19: error: reference not found
 * Platform enum for {@link PushDeviceRegistrationEntity#getPlatform()}.
```

--- a/pom.xml
+++ b/pom.xml
@@ -83,8 +83,6 @@
         <!-- Documentation Dependencies -->
         <springdoc-openapi-starter-webmvc-ui.version>2.6.0</springdoc-openapi-starter-webmvc-ui.version>
         <swagger-annotations-jakarta.version>2.2.22</swagger-annotations-jakarta.version>
-
-        <maven-javadoc-plugin.version>3.5.0</maven-javadoc-plugin.version>
     </properties>
 
     <dependencyManagement>
@@ -161,10 +159,38 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok-maven-plugin</artifactId>
+                <version>1.18.20.0</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.projectlombok</groupId>
+                        <artifactId>lombok</artifactId>
+                        <version>${lombok.version}</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <outputDirectory>${project.build.directory}/delombok</outputDirectory>
+                    <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
+                    <addOutputDirectory>false</addOutputDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>delombok</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
+                    <sourcepath>${project.build.directory}/delombok</sourcepath>
                     <failOnError>false</failOnError>
                 </configuration>
                 <executions>

